docs/src/main/asciidoc/datasource.adoc

```diff
@@ -3,62 +3,66 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Datasources
+[id="datasources"]
+= Get started with configuring datasources in Quarkus
 include::_attributes.adoc[]
-:categories: data
-:summary: With Quarkus, you can easily configure a datasource, or several if need be.
+:categories: data,getting-started
 
-Many projects that use data require connections to a relational database.
+Learn how to easily configure a Java Database Connectivity (JDBC) and Reactive datasource, their combination, or several datasources if required.
 
-The usual way of obtaining connections to a database is to use a datasource and configure a JDBC driver.
-But you might also prefer using a reactive driver to connect to your database in a reactive way.
+Many Java applications require connections to a relational database.
+The usual way to establish database connections is to combine a data source and a database driver.
```

Comment:
Sorry, one more thing:

Regarding the form 'datasource' without the space.  I like how consistent use of term is used, and the general term is mentioned in the introduction.

Would it help to add a note like this?

```suggestion
The usual way to establish database connections is to combine a data source and a database driver.

[Note]
====
The term _data source_ is more commonly referred to as _datasource_ in Quarkus throughout the interfaces and documentation.
====
```